src/typeDefs/inputs.ts

```diff
@@ -146,10 +146,10 @@ export const inputs = gql`
     description: String!
     location: String
     attendees: String
-    isPublic: Boolean!
-    visibleInSearch: Boolean!
     apiUrl: URL
     image: String
+    userRegistrationRequired: Boolean!
+    visibleInSearch: Boolean!
```

Comment:
both these fields don't need to be required in the input as database by default will assign them a sane default value, they're opt in fields